x-pack/plugins/cases/public/containers/use_get_cases.tsx

```diff
@@ -139,12 +140,18 @@ export interface UseGetCases extends UseGetCasesState {
 
 const empty = {};
 export const useGetCases = (
-  initialQueryParams: Partial<QueryParams> = empty,
-  initialFilterOptions: Partial<FilterOptions> = empty
+  params: Partial<{
```

Comment:
👍 

Do we need the outer `Partial` since each parameter is optional already?

Reply:
I think is needed because we could pass the `initialQueryParams` argument but with some of the `QueryParams` fields. For example `initialQueryParams: { perPage: 10 }`

Reply:
Yeah but I think that's already handled by doing this `Partial<QueryParams>` right?

I think we'd want Partial on the outer one if we had something like:

```
{
  initialQueryParams: Partial<Blah>;
  initialFilterOptions: Partial<Blah2>;
  b: string;
  c: string;
}
```

and we ant a quick way to make all of the fields in the object optional instead of adding a question mark to each.

Reply:
Oh you mean about the outer partial? Yeap, you are right.